### PR TITLE
add usage info to vagrant sub-project

### PR DESCRIPTION
**Pull request by johnmccabe**
adding some info on the project (following questions from @drigodwin this morning) and using it to test the new repo structure.

### CODE REVIEW

**Comment by johnmccabe** [comment]
pr#1 :cake: :tada: :balloon: :smiley:

File: vagrant/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#### Building the Vagrant archive
+
+This project is built as part of the main Brooklyn project but can also be built locally with:
+```
+mvn clean install
+```
+
+The resulting archive artifact can be found in:
+```
+./target/brooklyn-<BROOKLYN_VERSION>-dist.zip
```
Comment (aledsage) [inline vagrant/README.md]: Should this not be `./target/brooklyn-vagrant-$BROOKLYN_VERSION-dist.tar.gz`?

**Comment by aledsage** [comment]
@johnmccabe - looks good; only one minor comment to be addressed before merging.

**Comment by johnmccabe** [comment]
@aledsage fixed, thanks

**Pull request merged**